packages/core/src/storage/StorageUtility.spec.ts

```diff
@@ -444,33 +445,19 @@ describe("StorageUtility", () => {
   });
 });
 
-describe("loadOidcContextFromStorage", () => {
-  it("throws if no stored state matches the current request's", async () => {
-    const mockedStorage = mockStorageUtility({
-      "solidClientAuthenticationUser:mySession": {
-        issuer: "https://my.idp/",
-        codeVerifier: "some code verifier",
-        redirectUri: "https://my.app/redirect",
-        dpop: "true",
-      },
-    });
+describe("getSessionIdFromOauthState", () => {
+  it("returns undefined if no stored state matches the current request's", async () => {
```

Comment:
```suggestion
  it("returns undefined if we have no stored OIDC 'state' matching the current request's OIDC 'state' value", async () => {
```

Reply:
Resolved in 0ba475eb12e5452ea5a8427d6f1417b422d7fc4f